server.go

```diff
@@ -398,6 +385,22 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 func (s *Server) Start() {
 	log.WithField("version", VERSION).Info("Starting server")
 
+	if len(s.spanSinks) > 0 {
+		// Set up our internal trace client:
+		tb := &internalTraceBackend{}
+		var err error
+		s.TraceClient, err = trace.NewBackendClient(tb, trace.Capacity(200))
+		if err != nil {
+			panic(fmt.Sprintf("Could not set up internal trace backend: %v", err))
```

Comment:
nit: I think it'll be slightly more useful to have the "Could not set up..." message logged, and panic *just* on the error instead. The reason is that it's more flexible for the caller to `recover()` without having to parse the error string itself (which we want to avoid at all costs).